specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/databaseSecurityAlertPolicies.json

```diff
@@ -181,7 +181,6 @@
           "description": "Specifies the state of the policy. If state is Enabled, storageEndpoint and storageAccountAccessKey are required.",
           "type": "string",
           "enum": [
-            "New",
```

Comment:
I hadn't noticed before, but this is a pretty real breaking change. Is it just unused, or is there something nefarious about including this? If it's simply not useful, I'd recommend continuing to include it, and just have the server respond with an appropriate error message and documenting somewhere that this is deprecated.